README.md

```diff
@@ -78,7 +92,7 @@ soorgeon clean path/to/notebook.ipynb
 or
 
 ```
-soorgeon clean path/to/script.ipynb
```

Comment:
This doesn't work with .ipynbs?
We'd rather put this format since most users start with notebooks.

Reply:
It does. I think script means .py files? Otherwise, there already exists `soorgeon clean path/to/notebook.ipynb` and this seems redundant.

Reply:
Yeah I know, but this is different. I just don't understand why it was changed from .ipynb to .py?

Reply:
We mentioned `can apply black and isort for .ipynb and .py files`, but .ipynb is mentioned twice.

The previous version is the below, which seems redundant?
`soorgeon clean path/to/notebook.ipynb`
`soorgeon clean path/to/script.ipynb`

Reply:
Ok, yes that's a good catch, resolving!